src/consumer/consumerGroup.js

```diff
@@ -166,7 +166,7 @@ module.exports = class ConsumerGroup {
     }
 
     // Check if the consumer is aware of all assigned partitions
-    for (let topic of assignedTopics) {
+    for (let topic of keys(currentMemberAssignment)) {
```

Comment:
Can you assign `keys(currentMemberAssignment)` to something like `safeAssignedTopics`? This will help us in the future